gulpfile.js

```diff
@@ -44,10 +44,7 @@ gulp.task('watch', function() {
 // gulp.task('prepublish');
 
 gulp.task('ts', function() {
-  return gulp
-    .src('generators/')
-    .pipe(tsProject())
-    .js.pipe(gulp.dest('build'));
+  return gulp.src('generators/').pipe(tsProject());
```

Comment:
`'generators/` ? would not be `.src('generators/**/*.ts')`? Otherwise TS does not compiles anything.

Reply:
We dont have to compile the `templates` so I guess it has to be `generators/app/*.ts`.

Reply:
Yeah but you would need a pattern that is able to read all folders, in that case just use my patter + app to be more specific

Reply:
`'generators/**/*.ts'` I guess this will compile the templates too ? those typescripts templates.
cause they are in `generators/app/templates/typescript`. wouldn't it ?

Reply:
Ahh, it i is only 2 files then ... ?? Perhaps worth be more specific.

Reply:
yeah I guess

`['generators/app/index.ts','generators/app/type.ts']` ? this fits I guess !!

Reply:
Yeah, that's much better. At first glance I thought you were transpiling all of them.